modules/ai_drivers/settings.sqf

```diff
@@ -1,8 +1,9 @@
 // Array containing vehicles that will have option to spawn AI driver in them
```

Comment:
This file is effectively unmodified, just showing up as having made changes for some reason..